packages/Molecules/Form/__tests__/fieldRplSelect.test.js

```diff
@@ -128,7 +128,7 @@ describe('FieldRplSelect', () => {
     wrapper.find('.rpl-select__trigger').trigger('keyup.enter')
 
     expect(wrapper.find('.rpl-select__dropdown').isVisible()).toBeTruthy()
-    expect(wrapper.find('#single-select-drop-down__topic_a').classes()).toContain('rpl-select__listitem--selected')
+    expect(wrapper.find('#single-select-drop-down__topic__005fa').classes()).toContain('rpl-select__listitem--selected')
```

Comment:
I'd like to add data attributes for all form elements so we can use them across the board for testing - this works for this PR though